boards/common/nucleo64/Makefile.include

```diff
@@ -0,0 +1,5 @@
+# include nucleo common serial configuration
+include $(RIOTBOARD)/common/nucleo/Makefile.include.serial
```

Comment:
Its included in `Makefile.include` now, required to try compiling.

Reply:
yes, and I am not sure, but I think I fixed it in #8065 (at least I hope so...).